### PR TITLE
TNO-126 Update make commands

### PR DESCRIPTION
**Pull request by Fosol**
A number of port changes still needed to be made within Keycloak, and Nginx.

Make commands have been changed.  `kafka-up`, `core-up`, `service-up`, and `utility-up` have all been removed.  Instead an argument has been added to the `make up p={profile name}` which allows for profile selection.  By default it will use the `all` profile.

`make init` will now only start a minimum of services for the purpose of initialization, but does not run any of the features.  You will need to run subsequent `make up p={profile name}` commands to run appropriate features.  Or just run `make up` if your machine can handle running everything.

## Highlights

- Update make commands
- Fix Keycloak config
- Fix App config

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,7 +49,7 @@ setup: ## Setup local environment for development, generate configuration files.
 init: ## Initialize your local environment and start the core solution.
 	$(info Initialize your local environment and start the core solution)
 	@make setup
-	@make up
+	@make up p=init
```
Comment (Fosol) [inline Makefile:52]: Default only run the basic services that need to be initialized (i.e. database, kafka, elasticsearch, and keycloak)

File: Makefile [hunk 2]
```diff
@@ -63,119 +63,42 @@ nuke: ## Stop all containers, delete all containers, volumes, and configuration
...
+	$(info Stop, build, and start all containers or the one specified (n=$(n), p=$(p)))
+	@make stop n=$(n) p=$(p)
+	@make build n=$(n) p=$(p)
+	@make up n=$(n) p=$(p)
 
 remove: ## Remove all containers
 	$(info Remove all containers)
 	@docker-compose rm -sv
 
-##############################################################################
```
Comment (Fosol) [inline Makefile:97]: Replaced with `make up p=editor` or `make up p=subscriber`

File: Makefile [hunk 3]
```diff
@@ -63,119 +63,42 @@ nuke: ## Stop all containers, delete all containers, volumes, and configuration
...
-kafka-down: ## Stops the kafka containers and removes them
-	$(info Stops the kafka containers and removes them)
-	@docker-compose -f docker-compose.yml -f docker-compose.override.yml -f ./db/kafka/docker-compose.yml --profile kafka down -v
-
-kafka-rebuild: ## Build the kafka contains or the one specified (n=service name) and then start them after building
-	$(info Build the kafka contains or the one specified (n=service name) and then start them after building)
-	@make kafka-build n=$(n)
-	@make kafka-up n=$(n)
-
-##############################################################################
```
Comment (Fosol) [inline Makefile:134]: Replaced with `make up p=service`

File: Makefile [hunk 4]
```diff
@@ -63,119 +63,42 @@ nuke: ## Stop all containers, delete all containers, volumes, and configuration
...
-service-stop: ## Stops the service containers, or the one specified (n=service name)
-	$(info Stops the service containers, or the one specified (n=$(n)))
-	@docker-compose -f docker-compose.yml -f docker-compose.override.yml -f ./db/kafka/docker-compose.yml --profile service stop $(n)
-
-service-down: ## Stops the service containers and removes them
-	$(info Stops the service containers and removes them)
-	@docker-compose -f docker-compose.yml -f docker-compose.override.yml -f ./db/kafka/docker-compose.yml --profile service down -v
-
-##############################################################################
-# Utilities Management
```
Comment (Fosol) [inline Makefile:155]: Replaced with `make up p=utility`

File: auth/keycloak/README.md [hunk 5]
```diff
@@ -47,12 +47,6 @@ $ /opt/jboss/keycloak/bin/standalone.sh \
   -Djboss.management.http.port=7777
 ```
 
-Or (this doesn't appear to work on Windows)
```
Comment (Fosol) [inline auth/keycloak/README.md:50]: Not required

File: auth/keycloak/config/realm-export.json [hunk 6]
```diff
@@ -791,14 +791,14 @@
     "id" : "89ff6cf4-3755-4329-adab-ccfb74052c97",
     "clientId" : "tno-app",
     "name" : "Today's News Online",
-    "rootUrl" : "http://localhost:50080/app",
-    "baseUrl" : "http://localhost:50080/app",
+    "rootUrl" : "http://localhost:40080/app",
```
Comment (Fosol) [inline auth/keycloak/config/realm-export.json:794]: Fixed port in keycloak configuration.  You can manually update your local instance or you will need to reinitialize your keycloak container and database.

File: network/nginx/config/default.conf [hunk 7]
```diff
@@ -27,7 +27,7 @@ server {
     proxy_ssl_session_reuse off;
     proxy_set_header Host $http_host;
     proxy_cache_bypass $http_upgrade;
-    proxy_pass http://host.docker.internal:50005;
+    proxy_pass http://host.docker.internal:40081;
```
Comment (Fosol) [inline network/nginx/config/default.conf:30]: Fixed Nginx default configuration with latest port selection

**Review by sturufous — APPROVED** [review]

**Pull request merged**